### ISSUE TITLE
[Windows] 'Spawning language server with cmd' error

### ISSUE THREAD

**Issue by pidgeon777** [zbirenbaum/copilot.lua#21]
When launching `Neovim` with `Lunarvim` config, I obtain this warning notification at startup:

`Spawning language server with cmd: 'C:\nvim\data\lunarvim\site\pack\packer/opt/copilot.lua/index.js' failed with error message: UNKNOWN: unknown error`

and no `copilot` suggestions are given when modifying a buffer.

OS: Windows 10 Pro

Neovim version:

```lua
NVIM v0.7.0-dev+1275-g00effff56
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilato da runneradmin@fv-az158-203

Features: -acl +iconv +tui
See ":help feature-compile"

   file vimrc di sistema: "$VIM\sysinit.vim"
         $VIM di riserva: "C:/Program Files/nvim/share/nvim"

Run :checkhealth for more info
```

`packer.nvim` config:

```lua
    {
      "zbirenbaum/copilot.lua",
      event = {"VimEnter"},
      after = "nvim-cmp",
      requires = {"zbirenbaum/copilot-cmp"},
      config = function()
        local cmp_source = { name = "copilot", group_index = 2 }
        table.insert(lvim.builtin.cmp.sources, cmp_source)
        vim.defer_fn(function()
          require("copilot").setup {
            plugin_manager_path = require("lvim.utils").join_paths(get_runtime_dir(), "site", "pack", "packer"),
          }
          end, 100)
      end,
    },
```

**Comment by zbirenbaum** [comment]
Hi, this isn't an issue with the plugin. I think the problem is that the server is launched using index.js, which is an executable that calls /usr/bin/env node dist/agent.js 

For you, you will want to use the override settings in setup to pass a windows valid cmd which launches 'node \<path-to-copilot-plugin\>/copilot/dist/agent.js'

I would also highly, highly recommend using WSL to avoid issues like this if you can't install Linux over windows completely. It's quite literally a single command to install. 

Here are the docs: https://docs.microsoft.com/en-us/windows/wsl/install

**Comment by pidgeon777** [comment]
I would like to make use of wsl but it is not possible for me, at the moment.

I know it would be better to run everything in Linux, but I like to maintain my Windows Neovim config which is actually working really well, but encountering some issues with `copilot.lua`.

I'm more than willing to help debug issues that could allow us to run your great plugin in Windows.

For example, is there a way to know if the `copilot` LSP is really active? I'm asking this because it seems that no autocomplete suggestion is given even if the LSP is reported as active, I'm trying to understand right now where the error might reside.

**Comment by zbirenbaum** [comment]
Sorry for the delay I have been very busy. You can know if it is active by checking :LspInfo if it is working and started it will show as attached. If it is started but not working it will show as not-attached. If it hasn't started it will just not show up.

**Comment by zbirenbaum** [comment]
Ok, very sorry about how long it took for such a simple fix but this is a very busy time of year :( I fixed your other issue #22 about overrides not applying, so if you just input the correct cmd for your system this should no longer be a problem.

I'm gonna close this, but if you keep having issues feel free to reopen, as I do want to address this.

Also though, why is WSL not an option out of curiosity? These days its just one command to install.
